modules/persistent-storage-snapshots-starting-controller-provisioner.adoc

```diff
@@ -46,6 +46,12 @@ rules:
     resources: ["volumesnapshotdatas"]
     verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
 ----
+. As the cluster admin, provide the `hostNetwork security context` constraint (SCC):
+----
```

Comment:
This needs a "+" in the line above it, so that it is properly indented with step 2